src/helpers.jl

```diff
@@ -113,59 +113,73 @@ function override_pref(name, server_pref, override)
     server_pref
 end
 
-function fetch_misc_apis_versions(ctx::KuberContext; override=nothing, verbose::Bool=false)
+function fetch_misc_apis_versions(ctx::KuberContext; override=nothing, verbose::Bool=false, num_tries=1)
     apis = ctx.apis
-    vers = getAPIVersions(ApisApi(ctx.client))
-    api_groups = vers.groups
-    for apigrp in api_groups
-        name = apigrp.name
-        pref_vers_type = apigrp.preferredVersion
-        pref_vers_version = override_pref(name, pref_vers_type.version, override)
-        pref_vers = name * "/" * pref_vers_version
-        verbose && @info("$name ($(api_group(name))) versions", supported=join(map(x->x.version, apigrp.versions), ", "), preferred=pref_vers_version)
+    @repeat num_tries try
+        vers = getAPIVersions(ApisApi(ctx.client))
```

Comment:
wrapping just the `getAPIVersions` should be enough I think, that's the only place where a remote IO happens and IOError is a possiblity